compiler/luci/pass/src/CircleOptimizer.cpp

```diff
@@ -378,6 +379,11 @@ void CircleOptimizer::quantize(loco::Graph *g) const
     logo::PhaseRunner<logo::PhaseStrategy::Saturate> phase_runner{g};
     phase_runner.attach(&prog);
     phase_runner.run(phase);
+
+    // Verify the type/granularity of the quantized model
+    luci::QuantizedModelVerifyPass verifier(str_to_dtype(output_dtype),
+                                            str_to_granularity(granularity));
+    verifier.run(g);
```

Comment:
result is not used. is this OK?

Reply:
Ah.. verifier did not verify anything :frowning_face: I made verifier throw an exception when the verification fails.